src/SecTester.Scan/Target/Har/Response.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+
+namespace SecTester.Scan.Target.Har;
+
+public record Response(int Status, string StatusText, string HttpVersion, IEnumerable<Cookie> Cookies,
+  IEnumerable<Header> Headers, string RedirectUrl, int HeaderSize, int BodySize, Content Content)
+{
+  public string StatusText { get; } = StatusText ?? throw new ArgumentNullException(nameof(StatusText));
+  public string HttpVersion { get; } = HttpVersion ?? throw new ArgumentNullException(nameof(HttpVersion));
+  public string RedirectUrl { get; } = RedirectUrl ?? throw new ArgumentNullException(nameof(RedirectUrl));
+  public IEnumerable<Cookie> Cookies { get; } = Cookies ?? throw new ArgumentNullException(nameof(Cookies));
+  public IEnumerable<Header> Headers { get; } = Headers ?? throw new ArgumentNullException(nameof(Headers));
+  public Content Content { get; } = Content ?? throw new ArgumentNullException(nameof(Content));
+
+  public static readonly Response Default = new(200, "OK", "HTTP/1.1", new List<Cookie>(),
+    new List<Header>(), "", -1, -1, Content.Default);
```

Comment:
This is the default value for `ScanFactory`, but it has nothing in common with the default value for HAR spec in general